archetypes/blog-post.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: blog
+title: `<the title of the post>`
+date: <yyyy>-<mm>-<dd>
+slug: <seo-friendly-version-of-title-separated-by-dashes>
+---
+
+**Author:** <your name> (<your organization name>), <another author's name> (<their organization>)
+
+Instructions:
+- Replace these instruction and the following text with your content.
+- Replace `<angle bracket placeholders>` with actual values. For example, you would update `date: <yyyy>-<mm>-<dd>` to look something like `date: 2021-10-21`.
+- For convenience, use third-party tools to author and collaborate on your content.
+- To save time and effort in reviews, check the spelling, grammar, and style of your content before contributing.
+- Feel free to ask for assistance in the Kubernetes Slack channel, [#sig-docs-blog](https://kubernetes.slack.com/archives/CJDHVD54J).
```

Comment:
I'd put all this bit inside an HTML comment so it never gets rendered even in previews etc.

Reply:
👍

Reply:
Updated. Thanks.

Reply:
Updated. Thanks. What about the rest of the example content? I've left it bare for now.